app/Filament/Admin/Resources/NodeResource/Widgets/NodeStorageChart.php

```diff
@@ -39,20 +39,18 @@ class NodeStorageChart extends ChartWidget
 
     protected function getData(): array
     {
-        /** @var Node $node */
-        $node = $this->record;
+        $total = Number::format(($this->node->statistics()['disk_total'] ?? 0) / 1024 / 1024 / 1024, maxPrecision: 2);
+        $used = Number::format(($this->node->statistics()['disk_used'] ?? 0) / 1024 / 1024 / 1024, maxPrecision: 2);
 
-        $total = ($node->statistics()['disk_total'] ?? 0) / 1024 / 1024 / 1024;
-        $used = ($node->statistics()['disk_used'] ?? 0) / 1024 / 1024 / 1024;
         $unused = $total - $used;
 
         return [
```

Comment:
Set locale on the chart itself (see #832 for reference)

Reply:
Done